### PR TITLE
function to send chunked responses

### PR DESCRIPTION
**Pull request by lohanidamodar**
Send responses in multiple chunks. pass `$last` true to end the response

### CODE REVIEW

File: tests/e2e/server.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    Process::signal(2, function () use ($http) {
+        echo 'Stop by Ctrl+C';
+        $http->shutdown();
+    });
+});
+
+App::get('/')
+    ->inject('response')
+    ->action(function ($response) {
+        var_dump($response);
```
Comment (TorstenDittmann) [inline tests/e2e/server.php:61]: leftover.

File: tests/e2e/server.php [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            $response->chunk($word, $key == 1);
+        }
+    });
+
+$http->on('request', function (SwooleRequest $swooleRequest, SwooleResponse $swooleResponse) {
+    $request = new Request($swooleRequest);
+    $response = new Response($swooleResponse);
+
+    $app = new App('UTC');
+    // var_dump($request);
```
Comment (TorstenDittmann) [inline tests/e2e/server.php:79]: leftover.

**Review by TorstenDittmann — APPROVED** [review]

File: tests/e2e/server.php (hunk 1 above)
Comment (lohanidamodar) [reply tests/e2e/server.php]: removed

File: tests/e2e/server.php (hunk 2 above)
Comment (lohanidamodar) [reply tests/e2e/server.php]: removed

File: src/Swoole/Response.php [hunk 3]
```diff
@@ -95,6 +95,46 @@ public function send(string $body = '', int $exit = null): void
...
+     * Output response
+     *
+     * Generate HTTP response output including the response header (+cookies) and body and prints them.
+     *
+     * @param string $body
+     * @param bool $last
+     *
+     * @return void
+     */
+    public function chunk(string $body = '', bool $last = false): void
```
Comment (eldadfux) [inline src/Swoole/Response.php:108]: Any chance this should or could be part of utopia/fw i a way that we can keep a consistent interface?

Comment (eldadfux) [reply src/Swoole/Response.php:108]:
```suggestion
    public function chunk(string $body = '', bool $end = false): void
```

Might be more decoupled from the files behavior and a bit more generic.

**Review by eldadfux — CHANGES_REQUESTED** [review]

**Review by eldadfux — APPROVED** [review]
Waiting for the release of utopia/fw

**Review by eldadfux — APPROVED** [review]

**Pull request merged**